chiron/potential.py

```diff
@@ -272,9 +272,24 @@ def __init__(
         self,
         topology: Topology,
         k: unit.Quantity = 1.0 * unit.kilocalories_per_mole / unit.angstrom**2,
-        x0: unit.Quantity = 0.0 * unit.angstrom,
+        x0: unit.Quantity = jnp.array([[0.0, 0.0, 0.0]]) * unit.angstrom,
```

Comment:
We might also want to implement a `MultipleHarmonicOscillators` test system, which could just be `N` independent harmonic oscillators with a different spring constant for each particle based on the element. This is still analytically tractable.

We might want a `TestSystem` base class or mix-in that allows us to provide analytical properties to validate against, like in `openmmtools.testsystems`.

Reply:
This has been addressed in PR #15